dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/internal/SpringConstants.java

```diff
@@ -17,12 +17,12 @@
  */
 public final class SpringConstants {
     public static final String SPRING_APP_TYPE = "spr";
+    public static final String SPRING_LIB_CACHE_NAME = "lib.index.cache";
     public static final String SPRING_BOOT_SUPPORT_CAPABILITY = "spring.boot.support";
     public static final String SPRING_BOOT_SUPPORT_CAPABILITY_JARS = "jars";
-    private static final String SPRING_LIB_CACHE_DIR = "lib.index.cache/";
-    public static final String SPRING_SHARED_LIB_CACHE_DIR = WsLocationConstants.SYMBOL_SHARED_RESC_DIR + SPRING_LIB_CACHE_DIR;
+    public static final String SPRING_SHARED_LIB_CACHE_DIR = WsLocationConstants.SYMBOL_SHARED_RESC_DIR + SPRING_LIB_CACHE_NAME + "/";
```

Comment:
consider using File.separator in these since they are constants and may in the future be used in code paths where they are not normalized bu some other utility.

Reply:
These constants are currently used for com.ibm.wsspi.kernel.service.location.WsLocationAdmin.resolveResource(String) which takes a URI.  Using a '/' here is correct, even on Windows.